File: leetcode/1344.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        min_ang = 6 * minutes
        hour_ang = 30 * (hour % 12) + min_ang / 12
        diff = abs(hour_ang - min_ang)
        return min(diff, 360 - diff)

File: leetcode/1344.cpp
class Solution {
  public:
    double angleClock(int hour, int minutes) {
        double min_ang = 6. * double(minutes);
        double hour_ang = 30. * double(hour % 12) + min_ang / 12.;
        double diff = std::abs(hour_ang - min_ang);
        return std::min(diff, 360. - diff);
    }
};
